app/src/main/java/in/ac/bits_pilani/goa/ard/utils/Browser.java

```diff
@@ -0,0 +1,408 @@
+package in.ac.bits_pilani.goa.ard.utils;
+
+import android.app.Activity;
+import android.app.PendingIntent;
+import android.content.res.TypedArray;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Color;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.AnimRes;
+import android.support.annotation.ColorInt;
+import android.support.annotation.NonNull;
+import android.support.customtabs.CustomTabsIntent;
+import android.support.v4.app.ActivityCompat;
+import android.webkit.URLUtil;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+
+import in.ac.bits_pilani.goa.ard.R;
+
+import static in.ac.bits_pilani.goa.ard.utils.AHC.dpToPx;
+
+/**
+ * Browser opens web links.
+ * Includes default customizations (in constructor) which can be overridden.
+ *
+ * <h1>Usage</h1>
+ * <p>
+ * To open url with default customizations from fooActivity
+ * <pre>
+ * {@code
+ * new Browser(this).launchUrl("example.com");
+ * }
+ * </pre>
+ * </p>
+ * <p>
+ * To override values :
+ * <pre>
+ * {@code
+ * new Browser(this).setToolBarColor(ActivityCompat.getColor(R.color.cyan_500), this).launchUrl("example.com");
+ * }
+ * </pre>
+ * </p>
+ *
+ * @author Rushikesh Jogdand
+ */
+@SuppressWarnings("WeakerAccess")
+public final class Browser {
+
+    /**
+     * Key for title of menu item.
+     */
+    private static final String KEY_MENU_ITEM_TITLE = "browser.menuItems.title";
```

Comment:
@Rushi98 
Should these fields be shifted to a util class (like BrowserUtil) to prevent unnecessary cluttering?